jax/_src/scipy/stats/_rankdata.py

```diff
@@ -0,0 +1,84 @@
+# Copyright 2023 The JAX Authors.
```

Comment:
I think it would make sense to put this in `_core.py` rather than its own submodule. The reason all the distributions have their own submodules here is because the submodule structure is part of the API.